Src/Plugins/Security/SEC101_043.NuGetCredentialsValidator.cs

```diff
@@ -128,6 +128,11 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                         return ReturnUnauthorizedAccess(ref message, asset: host, account: id);
                     }
 
+                    case HttpStatusCode.PaymentRequired:
```

Comment:
No comment? Also, have we proven that this condition doesn't indicate a valid key + an expired subscription?